lib/util/DiagramWalkerUtil.js

```diff
@@ -92,6 +93,24 @@ export function participatesIn(participant, shape) {
   return getParticipants(shape).includes(participant);
 }
 
+/**
+ * Takes variable amount of types and checks if the shape is of any of these types and not a label.
+ * @param shape
+ * @param {...*} var_args
+ * @returns {boolean}
+ */
+export function isAnyAndNotLabel(shape, var_args) {
```

Comment:
What exactly is the purpose of `var_args`? Why not use the rest parameter notation?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters